CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/jdbc/JDBCSourcePane.java

```diff
@@ -167,9 +172,10 @@ public JDBCSourcePane(final JDBCImportController importController) {
                     dbConnectionComboBox.setItems(connections);
                 }
             });
-            connectionsPane.add(removeBtn, 2, 1, 1, 1);
+            addRemModGp.add(removeBtn, 2, 1, 1, 1);
+            connectionsPane.add(addRemModGp, 0, 1, 1, 1);
 
-            final Button buttonCancel2 = new Button(ACTION_CANCEL);
+            final Button buttonCancel2 = new Button("OK");
```

Comment:
actually, can you rename the variable here sine it is no longer cancel?

Reply:
@Atlas139mkm can you rename this to `buttonOk2` or something better?